Apps/W1/Shopify/app/src/Order handling/Tables/ShpfyOrderLineAttribute.Table.al

```diff
@@ -0,0 +1,40 @@
+/// <summary>
+/// Table Shpfy Order Line Attribute (ID 30123).
+/// </summary>
+table 30148 "Shpfy Order Line Attribute"
+{
+    Caption = 'Shopify Order Attributes';
+    DataClassification = CustomerContent;
+
+    fields
+    {
+        field(1; "Order Id"; BigInteger)
+        {
+            Caption = 'Order Id';
+            DataClassification = CustomerContent;
+        }
+        field(2; "Order Line Id"; Guid)
+        {
+            Caption = 'Line Id';
+            DataClassification = CustomerContent;
```

Comment:
I think these fields should habe `DataClassification = SystemMetaData;`

Reply:
ok, Done